src/main/kotlin/app/revanced/patches/remini/misc/premium/annotations/UnlockPremiumCompatibility.kt

```diff
@@ -0,0 +1,13 @@
+package app.revanced.patches.remini.misc.premium.annotations
+
+import app.revanced.patcher.annotation.Compatibility
+import app.revanced.patcher.annotation.Package
+
+@Compatibility(
+	[Package(
+		"com.bigwinepot.nwdn.international", arrayOf("3.7.46.202160002")
```

Comment:
As per [discussion](https://github.com/revanced/revanced-patches/pull/1299#discussion_r1051617601) this still needs to be fixed.